app/notebook/server/WebSockWrapper.scala

```diff
@@ -1,37 +1,40 @@
-package notebook
-package server
+package notebook.server
 
 import java.util.UUID
 
+import notebook.util.Logging
 import play.api.libs.iteratee._
 import play.api.libs.json._
 
-import notebook.util.Logging
-
 /**
  * Author: Ken
  */
 
 trait WebSockWrapper {
-  def session:String
-  def send(header: JsValue, session: String, msgType: String, channel:String, content: JsValue)
+
+  def session: String
+
+  def send(header: JsValue, session: String, msgType: String, channel: String, content: JsValue)
 }
 
-class WebSockWrapperImpl(sock: Concurrent.Channel[JsValue], val session:String) extends WebSockWrapper with Logging {
+class WebSockWrapperImpl(sock: Concurrent.Channel[JsValue],
+  val session: String) extends WebSockWrapper with Logging {
```

Comment:
if add `override def session: String = ???` method, deleting `val` will be ok. subclass must implements traint interface